modules/virt-extended-selinux-policies-for-virt-launcher.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-additional-security-privileges-controller-and-launcher.adoc
+
+[id="virt-extended-selinux-policies-for-virt-launcher_{context}"]
+= Extended SELinux policies for virt-launcher Pods
+
+The `container_t` SELinux policy for virt-launcher Pods is extended with the following rules:
+
+* `allow process self (tun_socket (relabelfrom relabelto attach_queue))`
+* `allow process sysfs_t (file (write))`
+* `allow process hugetlbfs_t (dir (add_name create write remove_name rmdir setattr))`
+* `allow process hugetlbfs_t (file (create unlink))`
+
+These additional rules provide virtual machines to take advantage of the following features:
+
+* Relabel and attach queues to its own TUN sockets, which is required to support network multi-queue. Multi-queue enables network performance to scale as the number of available vCPUs increases.
+
+* Allows virt-launcher Pods to write information to sysfs (`/sys`) files, which is required to enable Single Root I/O Virtualization (SR-IOV).
+
+* Read/write `hugetlbfs` entries, which is required to support huge pages. Huge pages are a method of managing large amounts of memory by increasing the memory page size.
```

Comment:
The intro sentence here is a bit weird. I'd suggest something like:

`With these rules in place, your virtual machines can:` 
or
`These rules enable the following virtualization features:`

But you also need to make the bullet points parallel in structure (and of course, match the intro sentence to whichever structure you choose).